config/initializers/catch_json_parser_errors.rb

```diff
@@ -1,3 +1,3 @@
 unless Rails.env.development?
-  Rails.application.config.middleware.insert_before ActionDispatch::ParamsParser, "CatchJsonParseErrors"
+  Rails.application.config.middleware.use CatchJsonParseErrors
 end
```

Comment:
`ActionDispatch::ParamsParser` is not a part of Rails middleware.

Available middleware

```
> rails middleware

use Rack::Sendfile
use ActionDispatch::Static
use ActionDispatch::Executor
use ActiveSupport::Cache::Strategy::LocalCache::Middleware
use Rack::Timeout
use Rack::Runtime
use Rack::MethodOverride
use ActionDispatch::RequestId
use Rails::Rack::Logger
use ActionDispatch::ShowExceptions
use WebConsole::Middleware
use ActionDispatch::DebugExceptions
use ActionDispatch::RemoteIp
use ActionDispatch::Reloader
use ActionDispatch::Callbacks
use ActiveRecord::Migration::CheckPending
use ActionDispatch::Cookies
use ActionDispatch::Session::CookieStore
use ActionDispatch::Flash
use Rack::Head
use Rack::ConditionalGet
use Rack::ETag
use Warden::Manager
use Bullet::Rack
use Rack::Deflater
run Wheel::Application.routes
```